src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -108,8 +114,8 @@
             </MultiBinding>
           </AutomationProperties.Name>
           <TabItem.Header>
-            <StackPanel Orientation="Horizontal">
-              <TextBlock x:Name="textInstalled" Text="{x:Static nuget:Resources.Label_Installed}"/>
+            <StackPanel Orientation="Horizontal" IsHitTestVisible="False" >
```

Comment:
This makes sense, but I'm curious what this solved for you? Or are you just doing it because it's logical to disable?
Can you still drill-in to see something like the Warning indicator?

Reply:
It avoids to focus the custom tab header elements (textInstalled text block or warning icon). If I set IsHitTestVisible=true, then, you can focus the textblock, the warning icon or the StackPanel, and, that avoid focusing on the TabItem

You need focus on TabItem to property trigger the color change.

Reply:
If the textblock and image controls are HitTestVisible, it also prevents the hover functionality from working on the TabItem because the hover is actually over the TextBlock or Image. The same applies for the click, and other Interactivity. So setting HitTestVisible to false makes them not steal the hover, click, etc. behavior from the TabItem.

Reply:
> It avoids to focus the custom tab header elements (textInstalled text block or warning icon)

Before this change, you couldn't focus on those individual controls. I just tested to be sure. 

> So setting HitTestVisible to false makes them not steal the hover, click, etc. behavior from the TabItem.

Sounds like it may have been needed for how you changed the hover styling trigger. In that case, makes sense.